README.md

```diff
@@ -70,7 +70,7 @@ git commit -m "Description of changes"
 ```bash
 git push --set-upstream origin feature-name
 ```
-1. Submit a pull request to the main branch of the [TheTechCommute](https://github.com/TheTechCommute/TheTechCommute)
+1. Submit a pull request to the main branch of the [https://github.com/TheTechCommute/techcommute](https://github.com/TheTechCommute/techcommute)
```

Comment:
Hi @Emmarie-Ahtunan. I don't think this change is necessary. We still want the text for the link to read as 'TheTechCommute'.